server/schedulers/hot_region_test.go

```diff
@@ -891,12 +896,12 @@ func (s *testHotReadRegionSchedulerSuite) TestWithPendingInfluence(c *C) {
 			hb.(*hotScheduler).clearPendingInfluence()
 
 			op1 := hb.Schedule(tc)[0]
-			testutil.CheckTransferLeader(c, op1, operator.OpLeader, 1, 3)
-			// store byte/key rate (min, max): (6.6, 7.1) | 6.1 | (6, 6.5) | 5
+			testutil.CheckTransferPeer(c, op1, operator.OpHotRegion, 1, 4)
+			// store byte/key rate (min, max): (6.6, 7.1) | 6.1 | 6 | (5,5.5)
```

Comment:
Why it changes, can you explain it?

Reply:
In the past, by transfer leader from 1 to 3 and move peer from 1 to 4, we obtained a distribution of 6.1,6.1,6.5,5.5; with this pr, we achieve a better distribution by moving peer from 1 to 4 twice, i.e. 6.1.6.1,6,6

Reply:
update